core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -144,7 +146,7 @@
             }
 
             normalizedPatternJoiner.add(':' + paramName);
```

Comment:
How about prepending `*` to `paramName` if `captureRestPathMatching` is true so that users easily see rest paths patterns from the startup logs produced by `Routers`.